arm-multilib/CMakeLists.txt

```diff
@@ -33,6 +33,11 @@ set(C_LIBRARY "picolibc" CACHE STRING "Which C library to use.")
 set_property(CACHE C_LIBRARY PROPERTY STRINGS picolibc newlib llvmlibc)
 set(LLVM_BINARY_DIR "" CACHE PATH "Path to LLVM toolchain build or install root.")
 set(LIBC_HDRGEN "" CACHE PATH "Path to prebuilt lbc-hdrgen if not included in LLVM binaries set by LLVM_BINARY_DIR")
+option(
+    ENABLE_QEMU_TESTING
+    "Tests using QEMU are enabled by default, but can be disabled."
```

Comment:
The help text, while correct, doesn't fit well with `ENABLE_QEMU_TESTING`. It would fit better with a DISABLE_QEMU_TESTING option. 

Perhaps
```
Enable tests that use QEMU. This option is ON by default.
```

If the text is changed, would be good to update the other places too.

Reply:
Done, and updated FVP options to match.